tests/tests/website_footer_tests.robot

```diff
@@ -0,0 +1,56 @@
+*** Settings ***
+Documentation  Smoke Tests
+Library        OperatingSystem
+Library        SeleniumLibrary
+Library        String
+Resource       ../src/page_objects/keywords/common.robot
+Resource       ../src/page_objects/keywords/home_page.robot
+Resource       ../src/page_objects/keywords/ubuntu.robot
+Resource       ../src/test_files/test_files.resource
+Library        ../src/page_objects/libraries/browser_lib.py
+Suite Setup   Prepare Test Environment
+Test Setup    Prepare For Test
+Test Teardown    Clean After Test
+
+
+*** Test Cases ***
+Find and open About page
+    [documentation]  Click button About and check if subpage is opened
+    [tags]  Regression
+    #TODO: Find button About, clicks it and verifies if a list with some text dislays
+    Click Element  ${AboutButton}
+    Wait Until Element Is Visible   ${AboutSubpageText}    1
+
+Find and open Known Issues page
+    [documentation]  Click button Known Issues and check if subpage is opened
+    [tags]  Regression
+    #TODO: Find button Known Issues, clicks it and verifies if a list with some text dislays
+    Click Element  ${KnownIssuesButton}
+    Wait Until Element Is Visible  //div[@id="issues"]/ul/*
+    
+Find and open Open source technologies page
+    [documentation]  Click button Open source technologies and check if subpage is opened
+    [tags]  Regression
+    #TODO: Find button Open source technologies, clicks it and verifies if a list with some text dislays
+    Click Element  ${OpenSourceTechnologiesButton}
+    Wait Until Element Is Visible  //div[@id="techs"]/ul/*
+
+Find and open Github repository page
+    [documentation]  Click button Github repository and check if subpage is opened
+    [tags]  Regression
+    Click Element       xpath=/html/body/div[2]/small/a[4]
```

Comment:
I think it could be moved to locators.py and maybe changed to be simillar to other variables from footer there?

As for "=" between word "xpath" and "/html/body/div[2]/small/a[4]" - maybe change it to ":" - as it appears to be preferred way.

From documentation "The explicit locator strategy is specified with a prefix using either syntax strategy:value or strategy=value. The former syntax is preferred because the latter is identical to Robot Framework's [named argument syntax](http://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#named-argument-syntax) and that can cause problems. Spaces around the separator are ignored, so id:foo, id: foo and id : foo are all equivalent."
As found in https://robotframework.org/SeleniumLibrary/SeleniumLibrary.html#Locating%20elements